### DISCUSSION TITLE
Preview function is broke

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#533] [0. General bugs and questions]
### Ask a question, start a conversation, or make an annoucement

After new changes, it seems the Preview function in published spaces is not working anymore. I keep getting this error and the changes do not reflect there. 
<img width="502" alt="Screenshot 2024-03-20 at 08 17 53" src="https://github.com/GitbookIO/community/assets/112613597/758118ee-a4af-49cd-aa4c-38284d772301">

Together with that, I wonder why there's no preview possibility for unpublished spaces. It kinda defeats the whole purpose of a preview for unpublished content.

**Comment by addisonschultz** [comment]
Hi @shimshim938! I believe this fix has been released. Can you try again and let us know?

**Reply by ahcsco** [reply]
Ok yeah. Working fine now. But why's it not available for unpublished spaces?

**Reply by addisonschultz** [reply]
I don't think the preview feature has been ever available for unpublished spaces! If you'd like to preview changes for private content, you can check out these publishing options: https://docs.gitbook.com/published-documentation/share/space-publishing
